generate_datafiles/compound_gene_datafiles.py

```diff
@@ -12,9 +12,17 @@
 compound_url = "https://raw.githubusercontent.com/dhimmel/drugbank/7b94454b14a2fa4bb9387cb3b4b9924619cfbd3e/data/drugbank.tsv"
 gene_url = "https://raw.githubusercontent.com/dhimmel/entrez-gene/a7362748a34211e5df6f2d185bb3246279760546/data/genes-human.tsv"
 cbg_url = "https://raw.githubusercontent.com/dhimmel/integrate/93feba1765fbcd76fd79e22f25121f5399629148/compile/CbG-binding.tsv"
-full_map_output_file = "data/compound_gene/compound_binds_gene/compound_gene_pairs_binds_full_map.csv"
-full_map_sen_count_file = "data/compound_gene/compound_binds_gene/compound_gene-pairs_binds_mapping.csv"
-final_output_file = "data/compound_gene/compound_binds_gene/compound_gene_pairs_binds.csv"
+crg_url = "https://raw.githubusercontent.com/dhimmel/lincs/bbc6812b7d19e98637b44373cdfc52f61bce6327/data/consensi/signif/dysreg-drugbank.tsv"
+
+full_map_output_file = "../compound_gene/compound_gene_pairs.csv"
+
+cbg_sen_count_file = "../compound_gene/compound_gene-pairs_binds_sen_count.csv"
+cug_sen_count_file = "../compound_gene/compound_gene-pairs_upreg_sen_count.csv"
+cdg_sen_count_file = "../compound_gene/compound_gene-pairs_downreg_sen_count.csv"
+
+final_cbg_output_file = "../compound_gene/compound_binds_gene/compound_gene_pairs_binds.csv"
+final_cug_output_file = "../compound_gene/compound_gene_pairs_upregulates.csv"
+final_cdg_output_file = "../compound_gene/compound_gene_pairs_downregulates.csv"
```

Comment:
Might be helpful to clean up file path names using:
base_dir = os.path.join(os.path.dirname(os.getcwd()), 'compound_gene')
file = os.path.join(base_dir, "...csv')